cmd/controller/run.go

```diff
@@ -197,6 +217,7 @@ func Run(opts Options, runLog logr.Logger) {
 	}
 
 	runLog.Info("Exiting")
+	server.Stop()
```

Comment:
do we need this? what kind of cleanup are we doing here?

Reply:
No not really. I just put in there for completeness, but because we are exiting, its not required.